ocsmesh/hfun/collector.py

```diff
@@ -808,6 +890,8 @@ def _create_big_raster(self, out_path):
 
         rast_hfun_list = [
             i for i in self._hfun_list if isinstance(i, HfunRaster)]
+        if len(rast_hfun_list) == 0:
+            return None
```

Comment:
If there are not rasters in the collector input list, then return None. Sanity checks need to be added downstream if the `big_raster` is none